### ISSUE TITLE
Add-Ons Menu Endless Loop

### ISSUE THREAD

**Issue by Grateous** [home-assistant/iOS#2253]
Model Name: IPhone 11 Pro Max 256gb
Software Version: iOS 16.1
App 2022.10.1 (2022.424)

**Home Assistant Core Version**
2022.11.2
**Describe the bug**
Add on menu back arrow to main menu cycles back into add on.
**To Reproduce**

Clicking out of add on cycles back into add on menu. Etc etc etc endless loop. You sometimes can reboot phone to fix. However only guaranteed way is to reinstall the companion app. Don't see this in safari or on pc

**Expected behavior**
Goes back to main menu
**Screenshots**
Na
**Additional context**
See issue after using Zigbee2mqtt

**Comment by loovanloon** [comment]
I can confirm:

On my iPhone using the HA Companion app (v 2022.10.1):

- the left back arrow in the top menu does not function anymore. Pressing it does not take me back to to the configuration menu, hence I cannot reach the main menu.
- pressing the menu back arrow repeatedly will only toggle me between the last two pages I have visited in the addon section.
- swiping the app off / out of iOS memory / stopping the app does not resolve the problem, because restarting the app brings you back straight into the addon section ( hotel California: you can never leave…)
- restarting iPhone did not resolve the problem

**Comment by Daedilus** [comment]
Same issue here as well. Force closing the app and reopening while in the add on menu will sometimes bring back the three bars in the upper left hand corner.

**Comment by Mariusthvdb** [comment]
tis is not an iOS app issue, but a generic Frontend issue, see https://github.com/home-assistant/frontend/issues/14355

please up that issue, because frontend team seems to have not checked this

**Comment by Dinth** [comment]
> this is not an iOS app issue, but a generic Frontend issue, see [home-assistant/frontend#14355](https://github.com/home-assistant/frontend/issues/14355)
> 
> please up that issue, because frontend team seems to have not checked this.

I dont think it is, that frontend issue has been fixed ages ago, but im still being pestered with the bug on the iOS app. Literally 25% of times i go to Zigbee2MQTT addon settings, i am getting stuck there and need to reinstall the app to "get out".

**Comment by pgale** [comment]
I'm seeing similar. I was in the attached screen and left it - came back a day later, opened the app and the back button wasn't working so I can't get back to the Home Screen etc. I also had this before in another blueprint info view. 
Closing the app doesn't work and only a reinstall of the app solves it temporarily. There should be a home button on EVERY view/screen I think. 
![image](https://github.com/home-assistant/iOS/assets/513044/a44c96c1-d040-4271-9740-c4f3af2cd14b)

**Comment by Daedilus** [comment]
Did you try swiping from left to right?

**Comment by pgale** [comment]
Doh! Now I feel stupid! I didn't know you could do that. That works and makes this issue less of a problem :)

**Comment by raffoul** [comment]
Same issue in 2025. If I leave the ios while I use an addon. A coupe of hours later, the back button does not work.
Thanks for the tip.
